doc/source/ray-core/miscellaneous.rst

```diff
@@ -321,10 +288,6 @@ Test workload:
      - Actor ready time
      - Total time
    * - 20k (10 actors / node)
-     - 5.8s
-     - 146.1s
-     - 151.9s
-   * - 80k (40 actors / node)
-     - 21.1s
-     - 583.9s
-     - 605s
+     - 14.5s
+     - 136.1s
+     - 150.7s
```

Comment:
It seems there has been some regression now. 
With lightweight resource broadcasting:
- https://console.anyscale-staging.com/o/anyscale-internal/interactive-sessions/job_2x1ieba1vmywwws5jj7nm94l78

```
  "many_nodes_actor_tests_20000": {
    "actor_launch_time": 14.521619386000111,
    "actor_ready_time": 136.1954462419999,
    "total_time": 150.717065628,
    "num_actors": 20000,
    "success": "1",
    "throughput": 132.69897417830586
  }
```
Without it:
- https://console.anyscale-staging.com/o/anyscale-internal/interactive-sessions/job_cexlazp2gw44qz1gvwkt9ut6va

```
  "many_nodes_actor_tests_20000": {
    "actor_launch_time": 15.74624523600005,
    "actor_ready_time": 424.108206549,
    "total_time": 439.85445178500004,
    "num_actors": 20000,
    "success": "1",
    "throughput": 45.46958640258564
  }
```


I think maybe the previous tests are using m5.xlarge machine. Or there actually is a big regression.